src/registrar/assets/sass/_theme/_uswds-theme-custom-styles.scss

```diff
@@ -26,3 +26,7 @@ i.e.
 p {
   color: color('blue-10v');
 }
+
```

Comment:
This doesn't seem as if it's related to the build scripts? It isn't a huge deal, but generally I would want every change that appears in a PR to be related somehow.

Reply:
That makes sense in general. For this case I did it to make sure the style change was actually applied when I ran the deploy script. So would you typically revert or just not commit the change?

Reply:
I would typically back out a change that I made just to test that something happens. I would either make another commit that undoes the change so that it doesn't show up as a change in the overall PR, or not commit it in the first place.